packages/engine-core/src/tasks/task.rnv.crypto.encrypt.ts

```diff
@@ -40,13 +40,16 @@ const generateRandomKey = (length: number) =>
 
 const initializeCryptoDirectory = async (c: RnvContext, sourceFolder: string) => {
     const configDir = path.join(sourceFolder, 'appConfigs');
+    const targetFile = 'renative.private.json';
     mkdirSync(sourceFolder);
     mkdirSync(configDir);
 
+    if (c.paths.project.configPrivateExists) {
+        copyFileSync(c.paths.project.configPrivate, path.join(sourceFolder, targetFile));
```

Comment:
@ElenaDiachenko considering that we are making copy of project based private file I'd go for inquirierPrompt if we find project renative.private.json in project.

inquirer message: `Found existing private config in your project. What to do next?`

and offer 3 options:

`> Move renative.private.json into encrypted folder (Recommended)`  
`> Copy renative.private.json into encrypted folder`
`> Skip copy and leave renative.private.json in current location`